airbyte-integrations/connectors/source-pexels-api/source_pexels_api/pexels_api.yaml

```diff
@@ -0,0 +1,145 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.pexels.com"
+    http_method: "GET"
+    request_options_provider:
+      request_parameters:
+        query: |
+          {{
+            config['query']
+            if options['name'] == 'photos_search' or options['name'] == 'videos_search'
+            else ''
+          }}
+        orientation: |
+          {{
+            config['orientation']
+            if options['name'] == 'photos_search' or options['name'] == 'videos_search'
+            else ''
+          }}
+        size: |
+          {{
+            config['size']
+            if options['name'] == 'photos_search' or options['name'] == 'videos_search'
+            else ''
+          }}
+        locale: |
+          {{
+            config['locale']
+            if options['name'] == 'photos_search' or options['name'] == 'videos_search'
+            else ''
+          }}
```

Comment:
Why not transfer this to the stream requester and overwrite the request_parameter for that stream?

Reply:
Done